I/IGLWrap/iglwrap/iglwrap.cpp

```diff
@@ -0,0 +1,288 @@
+// TODO: write conversion functions Julia ↔ C++
+#include <Eigen/Dense>
```

Comment:
Where is this code from? Did you write it, or did you copy it from somewhere else? What license is it under?

Reply:
Do you mean the comment (which is mine, and obsolete — those functions are just below), or the whole file (which is also mine, and this indeed deserves a short mention at the top of the file)?

Reply:
The whole file